Docs/MANUAL.md

```diff
@@ -0,0 +1,322 @@
+# Gordian Seed Tool Manual
+
+<a href="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-listing.jpeg"><img src="https://raw.githubusercontent.com/BlockchainCommons/GordianSeedTool-iOS/shannona-minimanual/images/st-listing.jpeg" align="right" width=250 style="border:1px solid black;"></a>
+
+**Gordian Seed Tool** is an iOS-based seed manager that is a reference app for the Gordian system. It allows you to safely and securely store your cryptographic seeds and to export public keys, private keys, and seeds as required.
```

Comment:
I'm maybe being a little niggly here, but overall I think we need a little more clarity here. Seed tool only stores seeds, which are never used as keys, solely used to derive keys. Seedtool does not store keys, it has to derive them every time from the seed, and you can briefly export them - some like SSKR will be different each time you export. You can export seeds, but we should recommend that you only do so rarely, if the app will accept derived keys, you should derive only the key. Later, we also can sign things and not even export a key.

Specific to this intro paragraph: "It allows you to safely and securely store your cryptographic seeds and to DERIVE AND export public keys,  private keys, etc. YOU CAN ALSO EXPORT A SEED TO SSKR SHARDS, UR-QR, OR LEGACY BIP-39 MNEMONIC WORDS.

Reply:
(1) At the moment, we don't have a macOS release yet. I've added it because I don't want to forget it in the future, but we should remember to pull that if for some reason we can't put out a macOS version.

Reply:
(2) Thanks. Updated with a version of this.